nvflare/private/fed/server/job_cmds.py

```diff
@@ -632,15 +638,37 @@ def _unzip_data(self, download_dir, job_data, job_id):
             unzip_all_from_bytes(workspace_bytes, workspace_dir)
         return job_id_dir
 
+    def _clean_up_download(self, conn: Connection, tx_id: str):
+        """
+        Remove the job download folder
+        """
+        job_download_dir = self.tx_path(conn, tx_id)
+        shutil.rmtree(job_download_dir, ignore_errors=True)
+
     def pull_file(self, conn: Connection, args: List[str]):
-        if len(args) != 3:
-            self.logger.error("syntax error: missing file name")
+        """
+        Args: cmd_name tx_id folder_name file_name [end]
+        """
+        if len(args) < 4:
+            self.logger.error("syntax error: missing tx_id folder_name file name")
             return
```

Comment:
From the user's perspective, they should not worry about tx_id when downloading?

They should just "download_job_file [job_id] [file_name]"
Otherwise we making things more complicated for them?

Reply:
Correct! The user never sees tx_id. The "pull_file" is an internal command issued by the admin client's "pull_folder" command (which is mapped to the server's download_job command). So the user only sees "download_job job_id [result_name]" command.

Reply:
got it, this is clear for "download_job job_id [result_name]" command.

So we DO NOT have a command for users to download just a specific file in a job?

Reply:
No we don't, and we never did.

Reply:
got it, sounds good, all clear to me now.